testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1513,6 +1513,11 @@
   end
 end
 
+When(/^I reboot the "([^"]*)" minion through SSH$/) do |host|
```

Comment:
You also had
```
  When(/^I reboot the "([^"]*)" if it is a SLE Micro$/)
```
 but having this new step might be better.

(BTW it existed in the past, we removed it, because it had become dead code at some point, if i remember correctly)

Reply:
You are right, I remember, too.

Reply:
That step actually is implemented as

```ruby
 if slemicro_host?(host)
    step %(I reboot the "#{host}" minion through the web UI)
  end
```

We can potentially re-arrange the steps to take another variable in order determine whether to do the reboot via UI or SSH. That will probably reduce their number as we can reuse some.
But having really small and specialized steps is something I don't mind